src/app/services/node-actions.service.ts

```diff
@@ -145,7 +153,7 @@ export class NodeActionsService {
           );
         } else {
           contentEntities.forEach(node => {
-            batch.push(this[`${action}NodeAction`](node.entry, selection.id));
+            batch.push(this.copyNodeAction(node.entry, selection.id));
```

Comment:
@DenysVuika, most probably this causes the issues with moving file. This line should remain unchanged.